tests/link/pytorch/test_extra_ops.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+import pytensor.tensor as pt
+from pytensor.configdefaults import config
+from pytensor.graph import FunctionGraph
+from pytensor.graph.op import get_test_value
+from tests.link.pytorch.test_basic import compare_pytorch_and_py
+
+
+def test_pytorch_CumOp():
+    """Test PyTorch conversion of the `CumOp` `Op`."""
+
+    # Create a symbolic input for the first input of `CumOp`
+    a = pt.matrix("a")
+
+    # Create test value tag for a
+    a.tag.test_value = np.arange(9, dtype=config.floatX).reshape((3, 3))
+
+    # Create the output variable
+    out = pt.cumsum(a, axis=0)
```

Comment:
Test `axis=None` and axis=tuple(...)` if supported by the original Op. If tuple is allowed make sure you have more dimensions (3,) and only test 2 of them. The axis can be parametrized (prod and add as well) instead of adding more conditions inside the test

Reply:
Tried this on the original Op.  `axis=tuple(...)` does not work and gives a `TypeError`
`axis=None` gives the output as a 1-D array

Reply:
The Op `__init__` doesn't seem to check explicitly for axes but it does assume it is either None or an int. Can we add a check and raise an explicit ValueError if it's not either?

Reply:
Checked again, there is no error if we use `axis=(0)`, pytorch also returns the same output.
The error only comes when there are more than 1 elements in the tuple (Even `np.cumsum` gives `TypeError` in this case).

We could try adding a check and raise, but would that be needed in other Op implementations?
Since this would be used as an example, it might be complicated if a check and raise is not needed for other implementations.

Reply:
`(0)` is `0`, not a tuple with a 0 inside it, it would have to be `(0,)` to be a tuple with a single element inside. Does it work with `(0,)`?

Reply:
No, it gives a TypeError

Reply:
Which is fine but probably gives a typeerror in an obscure place. We should raise already in the init method of the Op to save people time